scripts/events_module/event_filters.py

```diff
@@ -293,6 +293,9 @@ def _check_cat_age(cat, ages: list) -> bool:
 
     if cat.age in ages:
         return True
+    elif not isinstance(cat.age, str) and cat.age.value in ages:
```

Comment:
```suggestion
    elif not isinstance(cat.age, str) and cat.age.value in ages:
```
What is this piece of code intended to check for?

Reply:
I was checking for places where `age` is still a str, and it appeared that in ShortEvents the m_c and r_c parameters are passed into `__init__` as dicts that are pulled stragiht from the JSON file. That was my attempt at checking if cat.age is of type `CatAgeEnum` without importing it.
Shall I check that `cat.age.value in ages`, or do something else?

Reply:
I've removed that now actually.